com.unity.render-pipelines.high-definition/Documentation~/HDR-Output.md

```diff
@@ -2,99 +2,101 @@
-| **Hue Shift Amount**                 | A value that determines how much your HDR settings can change the hue of your content. When the value is 0, the tonemapper will try to preserve the hue of the content as much as possible, tonemapping only the luminance. However, some content might have been authored assuming that hue-shift will happen when all color channels are tonemapped independently. To recover the hue-shift behaviour this slider can be moved toward See note below for more information. |
-| **Detect Paper White**               | Enable this property if you want HDRP to use the Paper White value that the device communicates. In many cases, this value may be not accurate. It is best practice to implement a calibration menu for your application to allow for these situations. |
-| **- Paper White**                    | Paper White value for situations in which you do not use the display-provided value or the display does not provide a value. |
-| **Detect Brightness Limits**         | Enable this property if you want HDRP to use the minimum and maximum nit values that the device communicates. In some cases, this value may be incorrect. It is best practice to implement a calibration menu for your application to allow for these situations |
-| **- Min Nits**                       | The minimum brightness value for situations in which you do not use the display provided value or the display does not provide a value. |
-| **- Max Nits**                       | The maximum brightness value for situations in which you do not use the display-provided value or the display does not provide a value. |
+| **Neutral HDR Range Reduction Mode** | The curve that the Player uses for tone mapping. The options are:<br />- **BT2390**: The default. Defined by the [BT.2390](https://www.itu.int/pub/R-REP-BT.2390) broadcasting recommendations.<br />- **Reinhard**: A simple Tone Mapping operator.<br /><br />This option is only available when you enable **[Additional Properties](More-Options.html)**. |
+| **Hue Shift Amount**                 | The value determines the extent to which your content retains its original hue after you apply HDR settings. When this value is 0, the tonemapper attempts to preserve the hue of your content as much as possible by only tonemapping [luminance](Physical-Light-Units.html). |
+| **Detect Paper White**               | Enable this property if you want HDRP to use the Paper White value that the display communicates to the Unity Engine. In some cases, the value the display communicates may not be accurate. Implement a calibration menu for your application so that users can display your content correctly on displays that communicate inaccurate values. |
+| **- Paper White**                    | The Paper White value of the display. If you do not enable **Detect Paper White**, you must specify a value here. |
```

Comment:
There's an extra  - (dash) showing in the table in the rich diff for this property